src/components/common.js

```diff
@@ -267,7 +267,8 @@ const commonSelectProps = {
     indicatorSeparator: () => ({ display: 'none' }),
     dropdownIndicator: (base, { selectProps: { isClearable } }) => _.merge(base, { paddingLeft: isClearable ? 0 : undefined }),
     multiValueLabel: base => ({ ...base, maxWidth: '100%' }),
-    multiValueRemove: base => _.merge(base, { ':hover': { backgroundColor: 'unset' } })
+    multiValueRemove: base => _.merge(base, { ':hover': { backgroundColor: 'unset' } }),
+    placeholder: base => ({ ...base, color: colors.dark(0.8) })
```

Comment:
TOL: This base color can vary between site skins. Currently it still passes even in the Terra skin, which has the lightest version of this color, but we should keep this in mind going forward. In the interest of maintaining accessibility in general, it might be worth splitting the color palette into 'functional' vs 'brand' sections, and keeping the functional colors constant to make sure our base components are predictable.